hackathon_site/hardware/api_filters.py

```diff
@@ -0,0 +1,30 @@
+from django_filters import rest_framework as filters
+
+from hardware.models import Hardware
+from hardware.serializers import HardwareSerializer
+
+
+class NumberInFilter(filters.BaseInFilter, filters.NumberFilter):
+    pass
+
+
+class HardwareFilter(filters.FilterSet):
+    queryset = Hardware
+    serializer_class = HardwareSerializer
+
+    in_stock = filters.BooleanFilter(
+        label="In stock?", method="filter_in_stock", help_text="In stock?"
+    )
+
+    @staticmethod
+    def filter_in_stock(queryset, _, value):
+        if value is True:
+            return queryset.filter(quantity_available__gt=0)
+        else:
+            return queryset.filter(quantity_available__lte=0)
```

Comment:
You can merge in the develop branch and change these to use `quantity_remaining` instead of `quantity_available` now